changelog.txt

```diff
@@ -4,6 +4,8 @@ please update the categories "new features" and "bugfixes" before a pull request
 
 version 2.3.0-dev
 new features:
+- added module to create CR templates for shallow RNO-G channels
```

Comment:
General question: Why can this only make templates for shallow? Shouldn't deep templates work in the same way? Shouldn't we make something more general?

Reply:
I don't see why it should not work for the deep channels. However, I never tested it. I removed "shallow" from the changelog.